tests/test_singularity.py

```diff
@@ -13,6 +14,42 @@
 sys.argv = ['']
 
 
+@needs_singularity
+def test_singularity_pullfolder(tmp_path):
+    for directory, subdir, files in os.walk(os.getcwd()):
+        for image in files:
+            if image.startswith("debian"):
+                os.remove(os.path.join(directory, image))
+                print("removed image hiding away deep within!")
```

Comment:
Oh, lets not do this.

If it seems necessary, then change to the `tmp_path` (which is unique for every invocation of this function) and copy the CWL descriptions there before executing

Reply:
Haha, this never seemed like a great idea to me but I wanted to give it a try to see if debian.img was actually persisting. I will remove that bit and push the change. It won't break the pipeline.